newsfragments/41496.significant.rst

```diff
@@ -0,0 +1,5 @@
+Removed deprecated methods in airflow/utils/dates.py
+
+Methods removed:
+  * date_range (Use ``airflow.timetables`` )
```

Comment:
I don’t think timetables replace `date_range`, at least not functionality wise… Airflow uses them to replace its usage of the function, but if you use `date_range` in a very generic way (just for a list of datetimes), timetables would be a very awkward replacement. I would instead not make any suggestions at all since there’s no direct replacement, and if you really want the exact functionality, the best approach is to just copy the function somewhere you can import from yourself.

Reply:
I have removed the recommendation.